react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react"
+export default function ToggleButton ({text, color}) {
+  const [isClicked, setisClicked] = useState(false)
+  const handleClick = () => {
+    setisClicked(!isClicked);
+    console.log(isClicked);
+  }
+  if (!isClicked) {
```

Comment:
This if/else works, but it duplicates quite a bit of code -- the only difference between lines 9-16 and 19-26 is that backgroundColor is "white" in one case and `color` in the other.

See if you can figure out how to do this without all that code duplication.